deployment/troposphere/template_utils.py

```diff
@@ -11,20 +11,21 @@
     'us-east-1'
 ]
 EC2_AVAILABILITY_ZONES = [
-    'b',
+    'a',
```

Comment:
I don't understand the change from 'b' to 'a'. I see your commit message about them not translating across accounts. Does this mean that 'b' is not available in the staging account, but 'a' and 'd' are available in both?